tests/seta/conftest.py

```diff
@@ -0,0 +1,131 @@
+import os
+import shutil
+
+from django.utils import timezone
+import pytest
+
+from treeherder.seta.common import get_root_dir
+from treeherder.seta.models import JobPriority
+from treeherder.seta.update_job_priority import ManageJobPriorityTable
+
+
+@pytest.fixture(scope="function")
+def test_setup(request):
+    path = get_root_dir()
+    if os.path.exists(path):
+        shutil.rmtree(path)
+    yield
+
+
+@pytest.fixture
+def job_priority_table_manager():
+    return ManageJobPriorityTable()
+
+
+@pytest.fixture
+def single_job_priority():
+    return JobPriority(
+        testtype='some-test-1',
+        buildtype='opt',
+        platform='windows8-64',
+        priority=1,
+        timeout=5400,
+        expiration_date=timezone.now(),
+        buildsystem='taskcluster'
+    )
+
+
+DATA = [
+    {
+        'platform_option': 'debug',
+        'platform': 'windows8-64',
+        'build_system_type': 'buildbot',
+        'testtype': 'web-platform-tests-1'
+    }, {
+        'platform_option': 'opt',
+        'platform': 'linux32',
+        'build_system_type': '*',
+        'testtype': 'reftest-e10s-1'
+    }
+]
+
+
+@pytest.fixture
+def sanitized_data():
+    return DATA
+
+
+@pytest.fixture
+def all_job_priorities():
+    data1 = DATA[0]
+    data2 = DATA[1]
+    return [
+        JobPriority(
+            testtype=data1['testtype'],
+            buildtype=data1['platform_option'],
+            platform=data1['platform'],
+            buildsystem=data1['build_system_type'],
+        ),
+        JobPriority(
+            testtype=data2['testtype'],
+            buildtype=data2['platform_option'],
+            platform=data1['platform'],
+            buildsystem=data2['build_system_type'],
+        )
+    ]
+
+
+@pytest.fixture
+def db_map():
+    return {
+        ('reftest-e10s-1', 'opt', 'windows8-64'): {
+            'pk': None, 'build_system_type': '*'
+        },
+        ('web-platform-tests-1', 'debug', 'windows8-64'): {
+            'pk': None, 'build_system_type': 'buildbot'
+        }
+    }
+
+
+@pytest.fixture
+def runnable_jobs_data():
```

Comment:
Any tests written under tests/seta/* can make use of this data fixure by simply declaring it as one of the functions arguments; e.g.:
```py
def test_case_a(runnable_jobs_data):
```